### DISCUSSION TITLE
Can't Register Account

### DISCUSSION THREAD

**Discussion by Shirley77777** [farcasterxyz/protocol#166] [General]
I already paid the fee while can't upload avatar image in the profile setting page, then I clicked exit to figure it out, while now I can't register & get my account recovery, no FID, what should I do now?

**Comment by horsefacts** [comment]
Hi Shirley—this sounds like a recent bug in the Warpcast client that should be fixed now. I'd try again to upload your avatar. If you're still having trouble, you can email support@merklemanufactory.com. 

This discussion forum is for the Farcaster protocol, so I'm going to close this conversation. Please reach out to the support email if you need help with Warpcast.
